pkg/controllers/dashboardapi/controller.go

```diff
@@ -9,6 +9,14 @@ import (
 	"github.com/rancher/rancher/pkg/wrangler"
 )
 
+func Add(ctx context.Context, wrangler *wrangler.Context) error {
```

Comment:
I don't think we need a wrapper fucntoin of Add. We can instead make AddClusterRepos public and give it a comment.